### PR TITLE
jupyterlab-lsp 3.10.2

### PR DESCRIPTION
**Pull request by bollwyvl**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.
  * added `third-party-licenses.json`

Notes:
- bumped minimum lab to 3.3

### CODE REVIEW

**Comment by bollwyvl** [comment]
@krassowski thanks for the release. anything i might have missed?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -51,7 +51,7 @@ outputs:
         - python >=3.7
       run:
         - jupyter-lsp >={{ server_version }}
-        - jupyterlab >=3,<4
```
Comment (krassowski) [inline recipe/meta.yaml:54]: The 3.x branch is still tested against 3.1 on CI and tests are passing so I would relax the pin range if that's not a problem

Comment (krassowski) [reply recipe/meta.yaml:54]: Unless you need that for licences stuff?

**Review by krassowski — COMMENTED** [review]
Thank you!

File: recipe/meta.yaml (hunk 1 above)
Comment (bollwyvl) [reply recipe/meta.yaml:54]: no, 3.1 added the license stuff, and i see that's the [actual current pin](https://github.com/jupyter-lsp/jupyterlab-lsp/blob/v3.10.2/python_packages/jupyterlab_lsp/setup.cfg#L40)

Comment (bollwyvl) [reply recipe/meta.yaml:54]: [f257678](https://github.com/conda-forge/jupyter-lsp-feedstock/pull/43/commits/f257678b96ee13de82d72a09bae241e959860259)

**Pull request merged**